src/doc/src/guide/continuous-integration.md

```diff
@@ -21,6 +21,35 @@ will not fail your overall build. Please see the [Travis CI Rust
 documentation](https://docs.travis-ci.com/user/languages/rust/) for more
 information.
 
+### GitHub Actions
+
+To test your package on GitHub Actions, here is a sample `.github/workflows/ci.yml` file:
+
+```yaml
+name: Cargo Build & Test
+
+on:
+  push:
+
+jobs:
+  build_and_test:
+    name: Rust project - latest
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toolchain:
+          - stable
+          - beta
+          - nightly
+    steps:
+      - uses: actions/checkout@v2
+      - run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
+      - run: cargo build --verbose
+      - run: cargo test --verbose
+```
+
+This will test all three release channels, see [GitHub Actions documentation](https://docs.github.com/en/actions) for more information.
```

Comment:
Worth calling out that any failure in nightly will fail the build here, unlike the other examples.

See https://github.com/actions/toolkit/issues/399

Reply:
Though we could just update the `run` command to include `||:` as a suffix for nightly.

https://unix.stackexchange.com/questions/78408/which-is-more-idiomatic-in-a-bash-script-true-or

Reply:
Thanks @misalcedo, while I agree that is an issue, what I think might be good is if at some point there is a better Actions link that is specific to Rust CI, then we could point people there - but I'm  hesitant to add complexity to the basic script due to @alexcrichton's comments:

> I think that continue-on-error and fail-fast would be removed because we don't necessarily have a great reason for deviating from the GitHub Actions defaults.

> I think we should make it somewhat terse and not have too many extra bells and whistles.

These lead me to believe that we've reached an acceptable starting place for Rust users with the current version, and that we could improve this example once a better solution to [actions/toolkit#399 is](https://github.com/actions/toolkit/issues/399) is available?

I'm open to suggestions for how to briefly explain this that CI will fail if any of the matrix jobs fail (and nightly being the one that is likely to without user error), but I think I'm otherwise going to leave this as is, and we should re-visit if anything changes that could improve the basic example or if documentation emerges that I could link out to.

Reply:
I think it's fine to leave as a comment but otherwise for the copy/paste snippet I agree we shouldn't have it be too too complicated.

Reply:
I've added a note.